browser/resources/settings/brave_wallet_page/brave_wallet_browser_proxy.ts

```diff
@@ -144,6 +146,18 @@ export class BraveWalletBrowserProxyImpl implements BraveWalletBrowserProxy {
     return sendWithPromise('clearPinnedNft')
   }
 
+  getTransactionSimulationOptInStatusOptions() {
+    return new Promise<string>(resolve => {
+      chrome.braveWallet.getTransactionSimulationOptInStatusOptions(resolve)
+    })
+  }
+
+  isTransactionSimulationsFeatureEnabled() {
+    return new Promise<boolean>(resolve => {
+      chrome.braveWallet.isTransactionSimulationsFeatureEnabled(resolve)
+    })
```

Comment:
We shouldn't use extension API. Just send message like other examples and handle the message in browser/ui/webui/settings/brave_wallet_handler.cc.

Reply:
updated: aded764